pom.xml

```diff
@@ -97,23 +103,39 @@
             <artifactId>batik-css</artifactId>
             <version>1.14</version>
             <exclusions>
+                <!-- exclude this old version of commons-io as newer can be used -->
+                <exclusion>
+                    <groupId>commons-io</groupId>
+                    <artifactId>commons-io</artifactId>
+                </exclusion>
                 <!-- exclude this as batik-css has a dependency that uses an older commons-logging and we want to eliminate the convergence mismatch -->
                 <exclusion>
                     <groupId>commons-logging</groupId>
                     <artifactId>commons-logging</artifactId>
                 </exclusion>
             </exclusions>
         </dependency>
+        <dependency>
+            <groupId>commons-io</groupId>
+            <artifactId>commons-io</artifactId>
+            <version>${version.io}</version>
+        </dependency>
         <dependency>
             <groupId>org.slf4j</groupId>
             <artifactId>slf4j-api</artifactId>
             <version>${version.slf4j}</version>
         </dependency>
+        <dependency>
```

Comment:
We've been trying to avoid including ANY kind of logger directly in AntiSamy. That way users of AntiSamy can pick whatever they want and not have to exclude/avoid what we include. Can we eliminate including jcl-over-slf4j?

Reply:
jcl-over-slf4j is an api.  because commons-logging is used by a transient, it is logging to no where unless user implements that on concrete way which basically means excluding it and adding jcl-over-slf4j because that is the physical api of antisamy.  Thus the reason here is to ensure logs are not lost if someone doesn't see that.  They will understand slf4j api usage here and aligning the other usage properly ensures no matter what they end up logging to, it picks up automatically for the library using commons-logging.

Reply:
Its far easier to see this with m2e in Eclipse and looking at resolved dependencies and seeing where commons-logging was excluded in favor of this.